feature/navigation/drawer/src/main/kotlin/app/k9mail/feature/navigation/drawer/domain/entity/DisplayUnifiedFolder.kt

```diff
@@ -0,0 +1,8 @@
+package app.k9mail.feature.navigation.drawer.domain.entity
+
+data class DisplayUnifiedFolder(
```

Comment:
I know, but I tested how much effort it would be to add more folders and it didn't seem too complicated. Disclaimer if I understood the search query correctly.